src/utils/parse-comment/index.js

```diff
@@ -75,20 +77,21 @@ function parseAddComment(doc, action) {
         })
         .data()[0].text
 
-    // TODO: handle plurals (e.g. some said docs)
-    let contributions = doc
-        .match('#Contribution')
+    const contributions = doc
+        .normalize({ plurals: true })
+        .words()
+        // .match('#Contribution')
```

Comment:
Any reason for dropping this?

Reply:
I tried keeping it but since words like "tests" and any of the plural forms of `validContributionTypes` get dropped after going through `match('#Contribution')`.